cmd/editscopedsk.go

```diff
@@ -65,7 +65,7 @@ func (p *EditScopedSkParams) SetDefaults(ctx ActionCtx) error {
 	if err := p.AccountContextParams.SetDefaults(ctx); err != nil {
 		return err
 	}
-	p.SignerParams.SetDefaults(nkeys.PrefixByteAccount, true, ctx)
+	p.SignerParams.SetDefaults(nkeys.PrefixByteOperator, true, ctx)
```

Comment:
this works for a local environment. For ngs/etc where the account is signed by the account, this needs to be relaxed to be PrefixByteAccount I think. You can try by updating an NGS account and see that it adds correctly